### ISSUE TITLE
Creating new product in new database causes error

### ISSUE THREAD

**Issue by metas-dh** [metasfresh/metasfresh#1196]
### Is this a bug or feature request?
Bug
### What is the current behavior?
When creating a new product (e.g. in e2e) in a new database, you get an error, 
SaveErrorNotUnique - org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "m_product_value"
  Detail: Key (ad_client_id, value)=(1000000, 1000000) already exists.
#### Which are the steps to reproduce?
* Have a new db, e.g. for e2e on mf15
* in window Produkt, create a new one: set a name and select a category (value is empty and read-only)
* try to save: SaveErrorNotUnique, see above
* change the product name, save again: OK, product is saved with the value 1000001
* works fine with all further products you create, value will be 1000002, 1000003 etc.
### What is the expected or desired behavior?
No error when creating the first product, value 1000001 (or later, depending on the values already existing) should be set directly.

**Comment by teosarca** [comment]
That's happening because the AD_Sequence.CurrentNext is wrong for "DocumentNo_M_Product" sequence.

Check
```
select Name, AD_Client_ID, CurrentNext from AD_Sequence where Name='DocumentNo_M_Product'
```

and compare it with the max integer value of M_Product.Value.

**Comment by metasnw** [comment]
tested fix on 5.41 with fresh master db : reproduced bug
after applying the fix: creating product was fine

**Comment by metas-dh** [comment]
Results of IT1
tested in release

tested in weekly e2e: no more error when creating new product in fresh db: OK
